lib/services/sf.js

```diff
@@ -623,6 +625,18 @@ function StateAbstract(options) {
             // success flag is false, the operation we tried to perform failed
             if (body && !body.success) {
               var data = body.data;
+              const auth = getAuthenticator(connectionConfig, httpClient);
+
+              if (body.code === '390195' && data.authnMethod === 'TOKEN') {
```

Comment:
Can we extract this magic number? (390195)

Reply:
I think this number is necessary. If I remove it, it impacts other token authentication or workflow. When I remove it and run a unit testing, the testing is stuck. I tried other properties from the response, but the code '390195) is unique.

Reply:
Okay, it is fixed.